docs/csharp/index.md

```diff
@@ -4,12 +4,37 @@ description: Learn how the C# Guide can help you gain extensive knowledge on C#,
 keywords: .NET, .NET Core
 author: BillWagner
 
-ms.date: 08/03/2016
+ms.date: 08/23/2017
 ms.topic: article
 ms.prod: .net
 ms.technology: devlang-csharp
 ms.devlang: csharp
 ms.assetid: 52db8280-0e53-40cf-858b-e8eef3997dea
+dev_langs: 
```

Comment:
I'm actually removing these from topics. Same for ms.devlang. They'll be set at a global level, unless the topic needs something different than the generic values. But C# will be the value for the csharp folder. The changes are sitting on a branch and I need to test them.